tools/scalar-schema/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM clojure:openjdk-11-lein-slim-buster AS BUILD
+COPY . /code
+WORKDIR /code
+RUN lein uberjar
+
+FROM openjdk:11-jre-slim
+WORKDIR /app
+COPY --from=BUILD /code/target/scalar-schema-standalone-*-SNAPSHOT.jar ./app.jar
+
+# ENV variables
+ENV SCALAR_DB_STORAGE cassandra
+ENV SCALAR_DB_HOST localhost
+ENV SCALAR_DB_USERNAME user
+ENV SCALAR_DB_PASSWORD pass
+ENV SCALAR_DB_SCHEMA schema.json
+ENV SCALAR_DB_REPLICATION_FACTOR 1
+
+# Run command
+ENTRYPOINT java -jar app.jar --$SCALAR_DB_STORAGE -h $SCALAR_DB_HOST -u $SCALAR_DB_USERNAME -p $SCALAR_DB_PASSWORD -f $SCALAR_DB_SCHEMA -R $SCALAR_DB_REPLICATION_FACTOR
```

Comment:
It looks like it supports only the Cassandra options. Don't we support the other adapters (Cosmos DB, DynamoDB, JDBC)?

Reply:
Actually, it is supported by the `--$SCALAR_DB_STORAGE` part. However, there are a few options I missed that need to be passed on to make it fully work. Like region, Cassandra port, etc

I noticed that the scalardl-schema-loader does not bother with ENV variables and just used the commands option for Docker. 

Like this

```
 command:
      - "--cassandra"
      - "-h"
      - "cassandra"
      - "-R"
      - "1"
```

I guess maybe it's easier to keep it like this.

Reply:
I have removed the usage of ENV variables and just use commands (when called from docker compose).  So now everything is supported.